example/model_usage.ipynb

```diff
@@ -503,7 +503,7 @@
    "source": [
     "Use the new prediction pipeline to run a prediction for the previously loaded test image.\n",
     "\n",
-    "The prediction pipeline returns a `Sample` object, which will be displayed via napari."
+    "The prediction pipeline returns a `Sample` object, which will be displayed."
```

Comment:
This relates to #398
Maybe we move changes to the model_usage notebook to yet another PR to adress some of the outstanding issues with it.